docs/source/reference/cli_fundamentals.rst

```diff
@@ -16,10 +16,37 @@ The dsgrid CLI commands are hierarchical with help at every level. For example:
    $ dsgrid registry
    $ dsgrid registry --help
 
+   $ dsgrid registry projects
+   $ dsgrid registry projects --help
+   $ dsgrid registry projects register --help
+
    $ dsgrid query
    $ dsgrid query --help
    $ dsgrid query project run --help
 
+Registry CLI Commands
+---------------------
+The ``dsgrid registry`` command group has a subgroup for each type of configuration stored in the
+registry: ``projects``, ``datasets``, ``dimensions``, and ``dimension-mapppings``.
+
+Each of those subgroups has four main commands:
+
+- ``register``: Register a new configuration item in the registry. This command typically takes a
+  JSON5 file as an input. After registration, all other commands will refer to the item by its ID.
+  Projects and datasets have a user-created string as their ID (``project_id`` and ``dataset_id``)
+  stored in the JSON5 file. Dimensions and dimension mappings receive integer IDs from the registry
+  during registration.
+- ``list``: List configuration items stored in the registry in a table. Each command offers the
```

Comment:
Change "configuration items" to just "items"? Or maybe change to "item configurations"

Reply:
I used configuration item because we refer "project config" and "dataset config" all over the place. I could be ok with just item. Document is another option. That is what the db uses.

Reply:
To me it seems most natural to just talk about "items," i.e., projects, datasets, etc. that are registered with dsgrid. 

I get that, for example, only the dataset configuration is available on the command line through the registry (and that, for example, the actual data is not so near at hand), but I think for users that's all just one thing, i.e., the "dataset", that is "in dsgrid" and by browsing the registry they are browsing the "dataset," not its "configuration" or "document". Does that make sense/seem right?

Reply:
I'll go with that. We may need to revisit the documentation where we often talk about "project config" and "dataset config." The documentation should probably focus most on the "dataset contributor" users and be consistent with terms. It makes sense to "go look at the project for dataset requirements" instead of "go look at the project config".